### PR TITLE
Defib requires armor to be off to use + medical Locale fixes

### PR DESCRIPTION
**Pull request by Dutch-VanDerLinde**
## About the PR
some medical locale now respects pronouns
defib can only be used if armor is off

## Why / Balance
CM13 parity
fixes defib not having RMC battery

## Media
image

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**
:cl:
- tweak: Defibrillating someone now requires their armor to be off.

### CODE REVIEW

File: Content.Server/Medical/DefibrillatorSystem.cs [hunk 1]
```diff
@@ -116,7 +118,15 @@ public bool CanZap(EntityUid uid, EntityUid target, EntityUid? user = null, Defi
         if (TryComp(target, out CMDefibrillatorBlockedComponent? block))
         {
             if (user != null)
-                _popup.PopupEntity(Loc.GetString(block.Popup), uid, user.Value);
+                _popup.PopupEntity(Loc.GetString(block.Popup, ("target", target)), uid, user.Value);
+            return false;
+        }
+
+        if (_inventory.TryGetSlotEntity(target, "outerClothing", out var armor)
+            && TryComp(armor, out CMDefibrillatorBlockedComponent? comp))
```
Comment (DrSmugleaf) [inline Content.Server/Medical/DefibrillatorSystem.cs:126]: This check should be done by using _inventory.GetSlotEnumerator(target, SlotFlags.OUTERCLOTHING) instead and checking if any of the items in those slots have the component

File: Content.Server/Medical/DefibrillatorSystem.cs [hunk 2]
```diff
@@ -144,13 +154,13 @@ public bool TryStartZap(EntityUid uid, EntityUid target, EntityUid user, Defibri
...
-                BreakOnMove = !component.AllowDoAfterMovement,
-                DuplicateCondition = DuplicateConditions.SameEvent,
-            });
+        {
+            BlockDuplicate = true,
+            BreakOnHandChange = true,
+            NeedHand = true,
+            BreakOnMove = !component.AllowDoAfterMovement,
+            DuplicateCondition = DuplicateConditions.SameEvent,
+        });
```
Comment (DrSmugleaf) [inline Content.Server/Medical/DefibrillatorSystem.cs:163]: We usually leave upstream files formatting unchanged for less merge conflicts
```suggestion
            {
                BlockDuplicate = true,
                BreakOnHandChange = true,
                NeedHand = true,
                BreakOnMove = !component.AllowDoAfterMovement,
                DuplicateCondition = DuplicateConditions.SameEvent,
            });
```

**Review by DrSmugleaf — CHANGES_REQUESTED** [review]
Thanks for the PR!

**Comment by Dutch-VanDerLinde** [comment]
requested changes done

**Review by DrSmugleaf — APPROVED** [review]

**Pull request merged**